core/src/main/java/org/elasticsearch/index/query/MultiMatchQueryBuilder.java

```diff
@@ -230,6 +254,11 @@ public MultiMatchQueryBuilder prefixLength(int prefixLength) {
         return this;
     }
 
+    public MultiMatchQueryBuilder prefixLength(Integer prefixLength) {
+        this.prefixLength = prefixLength;
```

Comment:
Note if we use defaults instead of null, then we have to always write them and render them. From a methodology perspective I think we should always try to find the defaults in the actual query whenever they exist and not use null.